src/terraform-provider-signalform/signalform/time_chart.go

```diff
@@ -374,13 +398,27 @@ func getSingleAxisOptions(axisOpt map[string]interface{}) map[string]interface{}
 			item["highWatermark"] = val.(float64)
 		}
 	}
+	if val, ok := axisOpt["high_watermark_label"]; ok {
```

Comment:
I think it's safe to do:
if val, ok := axisOpt["high_watermark_label"]; ok {
    item["highWatermarkLabel"] = val.(string)
}